src/components/WatchVideo.vue

```diff
@@ -329,6 +357,34 @@ export default {
             this.subscribed = !this.subscribed;
         },
         handleScroll() {
+            if (this.loading || !this.isPlaylist || !this.playlist.nextpage) return;
+            if (window.innerHeight + window.scrollY >= document.body.offsetHeight - window.innerHeight) {
```

Comment:
This would potentially require the user to scroll to the end of the page, instead of just till the end of the playlist.